Snakefile

```diff
@@ -108,6 +108,19 @@ rule prepare_scalars:
     shell:
         "python {input.script} {input.raw_scalars} {output}"
 
+rule prepare_heat_demand:
+    input:
+        weather="raw/weatherdata",
```

Comment:
```suggestion
        weather="raw/ERA5_weatherdata",
```

Reply:
Shall we change the name here or in the data?

Reply:
I would prefer to change it in the data, because the functionality of the script is not dependent on the data source.

Reply:
Ok!